### DISCUSSION TITLE
GP3-xTB code complexity

### DISCUSSION THREAD

**Discussion by tobigithub** [grimme-lab/xtb#1001] [General]
Will the new [GP3-xTB](https://www.mn.uio.no/hylleraas/english/news-and-events/events/joint-seminars/2024/Stefan-Grimme.html) include "just" a new [parameter file](https://github.com/grimme-lab/xtb/blob/main/param_gfn0-xtb.txt) or will it also require a major overhaul of the XTB code, including new extensions and many novel sub-routines? Thank you.

**Comment by marcelmbn** [comment]
It will include many new Hamiltonian contributions and a different type of basis set, among many other differences to GFNn-xTB. Thus, it comes with a totally new implementation which will come solely in [tblite](https://github.com/tblite/tblite). For further questions, just reach out to @thfroitzheim and myself.

**Comment by tobigithub** [comment]
Thank you @marcelmbn !
